gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/installation/EnvironmentResource.java

```diff
@@ -119,6 +120,11 @@ public PortalNavigationItemsResource getPortalNavigationItemResource() {
         return resourceContext.getResource(PortalNavigationItemsResource.class);
     }
 
+    @Path("/portal-page-content")
```

Comment:
Resource names are in plural form.
https://gravitee.slab.com/posts/gravitee-rest-api-guidelines-8ta6qkxl#h0i2u-case

```suggestion
    @Path("/portal-page-contents")
```